socorro/external/es/connection_context.py

```diff
@@ -103,14 +106,17 @@ def connection(self, name=None, timeout=None):
             elasticsearch.Elasticsearch(
                 hosts=self.config.elasticsearch_urls,
                 timeout=timeout,
-                connection_class=elasticsearch.connection.RequestsHttpConnection
+                connection_class=elasticsearch.connection.RequestsHttpConnection,
+                verify_certs=True
```

Comment:
A comment noting that this implies `use_ssl=True` would be useful for future readers.

Reply:
Mmm... Adding `verify_certs=True` doesn't imply anything. It just means that in the case of https, the elasticsearch-py library will set up requests with `verify_certs`.

Maybe I misunderstand your suggestion?

Reply:
Ah, no, it looks like I misunderstood your explanation. :+1: